pyproject.toml

```diff
@@ -87,7 +87,7 @@ default-groups = [
     "botocore-dev",
     "dev",
 ]
-required-version = ">=0.6.15"
+required-version = ">=0.7.0"
```

Comment:
safer would be like `~=0.7` or if more paranoid `~=0.7.0`

Reply:
Paranoid? Count me in :-)